content/en/docs/reference/kubernetes-api/ref/authentication/tokenrequest-v1.md

```diff
@@ -0,0 +1,161 @@
+---
+api_metadata:
+  apiVersion: "authentication.k8s.io/v1"
+  import: "k8s.io/api/authentication/v1"
+  kind: "TokenRequest"
+collapsible: false
```

Comment:
What does "collapsible" correspond to?

Reply:
nit: How were the weights applied?

Reply:
The order of Resources in each section comes from the ToC defined in this file: https://github.com/feloy/kubernetes-resources-reference/blob/master/config/v1.19/toc.yaml#L116